### DISCUSSION TITLE
Regarding Forcing perturbation frequency, State perturbation frequency and Observation perturbation frequency options

### DISCUSSION THREAD

**Discussion by Gayathridevi123** [NASA-LIS/LISF#1139] [Q&A]
Hi LIS Community,

I'm trying out the data assimilation simulations. So can anyone please let me know:-

1. What are Forcing perturbation frequency, State perturbation frequency and Observation perturbation frequency options in the LIS DA config file and how these values are given?
2. Also what is tcorr, xcorr and y corr values?
3. I'm trying with **GDAS+Noah+ESA CCI** and writing **OL surface model output as daily** ("1da"). So can you please suggest values for Forcing perturbation frequency, State perturbation frequency and Observation perturbation frequency options as well as tcorr, xcorr and ycorr?  

Any suggestions would be greatly appreciated.

Thank you
Gayathri

**Comment by karsenau** [comment]
Hi @Gayathridevi123:

You can take a look at some settings that we have used in the "pertattribs" tables, e.g., for forcing:
lis/configs/557WW-7.3-FOC/attribs/forcing_pertattribs.txt

These examples should give you some ideas.  I would recommend reading papers by Drs. Sujay Kumar, Rolf Reichle, and many others who publish using the EnKF and what values that they have applied. Many papers will offer such details.  

Also with the 1D EnKF type DA we apply in LIS, we would recommend trying some different perturbation frequency options with your setup to see what uncertainty that you may obtain or what could be "optimal" for your forcing dataset(s), states and region that you are working in (e.g., hot/dry region vs. cold/snowy region).

**Reply by Gayathridevi123** [reply]
Thank you @karsenau for your suggestion.
